core/simulations/scripts/warpTime.ts

```diff
@@ -0,0 +1,5 @@
+import { warpTime } from '../../helpers/hardhat';
+
+export default async () => {
+    await warpTime('custom');
+};
```

Comment:
the initial idea is that all the scripts here do not require the arguments. so this just wraps the function so that it provides the necessary args and the config is abstracted from them - it only has to know the name of the script and nothing more.

Reply:
removed the older warptime function from the store , now we can add it in the simulation instead.

Reply:
> the initial idea is that all the scripts here do not require the arguments

I think the initial idea was to create functions already in a way what have unnecessary/default parameters set as defaults. But I don't see why do we want to create new function just to wrap another function with default parameters (which has exactly the same name)

In my opinion you can use `warpTime` function directly in the simulation config